src/audio/mux/mux.c

```diff
@@ -279,6 +280,29 @@ static int demux_process(struct processing_module *mod,
 	return 0;
 }
 
+static int demux_trigger(struct processing_module *mod, int cmd)
```

Comment:
Should this be a separate commit ?

Reply:
@lgirdwood in principle it's all for the same goal - removing unused data and code from modular builds. This hunks don't change any code, they just move it around. Alternatively I could leave those functions where they are and just add `#if` - `#endif` to each of them but that would be too ugly. So, if allowed, I'd leave this as is, if however there's a strong desire for splitting this, I can do that too